grabbit/src/main/groovy/com/twcable/grabbit/client/batch/ClientBatchJob.groovy

```diff
@@ -42,8 +42,9 @@ class ClientBatchJob {
     public static final String WORKFLOW_CONFIGS = "workflowConfigIds"
     public static final String HOST = "host"
     public static final String PORT = "port"
-    public static final String USERNAME = "username"
-    public static final String PASSWORD = "password"
+    public static final String SERVER_USERNAME = "serverUsername"
+    public static final String SERVER_PASSWORD = "serverPassword"
+    public static final String CLIENT_USERNAME = "clientUsername"
```

Comment:
As an example, consider the case where the client is your local development instance and the server is production. You local instance is probably admin/admin, but the production server had better not be using those credentials... So the client needs to know the name of the user to write to its JCR as, and the credentials to use when connecting to the server for retrieving the data.